docs/02-app/02-api-reference/02-file-conventions/route-segment-config.mdx

```diff
@@ -197,6 +200,23 @@ Support for `preferredRegion`, and regions supported, is dependent on your deplo
 > - If a `preferredRegion` is not specified, it will inherit the option of the nearest parent layout.
 > - The root layout defaults to `all` regions.
 
+### `maxDuration`
```

Comment:
Which version of Next.js was this introduced in? That should be documented.

Reply:
Good point, I will add that tomorrow